test/unittests/execute_call_test.cpp

```diff
@@ -286,10 +288,12 @@ TEST(execute_call, imported_functions_call_indirect)
     ASSERT_EQ(module.codesec.size(), 2);
 
     constexpr auto sqr = [](Instance&, span<const Value> args, int) {
-        return Value{args[0] * args[0]};
+        const auto x = as_uint32(args[0]);
+        return Value{uint64_t{x} * x};
     };
     constexpr auto isqrt = [](Instance&, span<const Value> args, int) {
-        return Value{(11 + args[0] / 11) / 2};
+        const auto x = as_uint32(args[0]);
+        return Value{(11 + uint64_t{x} / 11) / 2};
```

Comment:
These two (sqr, isqrt) expect i64 as output, should we explicitly cast them?

Reply:
No. This is fine.